docs/reference/edot-collector/components/hostmetricsreceiver.md

```diff
@@ -0,0 +1,159 @@
+---
+navigation_title: Host metrics receiver
+description: The host metrics receiver is an OpenTelemetry Collector component that collects system-level metrics from the host machine.
+applies_to:
+  stack:
+  serverless:
+    observability:
+  product:
+    edot_collector:
+products:
+  - id: elastic-agent
+  - id: observability
+  - id: edot-collector
+---
+
+# Host metrics receiver
+
+The host metrics receiver collects system-level metrics from the host machine, including CPU usage, memory, disk I/O, network traffic, filesystem statistics, and process information. It provides essential infrastructure monitoring data that powers {{product.observability}}'s [Infrastructure dashboards](docs-content://reference/observability/observability-host-metrics.md).
+
+The receiver uses a set of specialized scrapers to gather metrics from different subsystems, making it flexible enough to collect only the metrics you need.
+
+## Default usage in EDOT
+
+The `hostmetricsreceiver` is included by default in the EDOT Collector for both standalone and Kubernetes deployments.
+
+### Standalone agent mode
+
+In standalone deployments, the host metrics receiver is part of the default [host metrics collection pipeline](../config/default-config-standalone.md#host-metrics-collection-pipeline). It collects system metrics at 60-second intervals and can be configured to export either:
+
+- Directly to {{es}} using the `elasticsearch` exporter (with ECS translation through the `elasticinframetrics` processor).
+- To the [{{motlp}}](opentelemetry://reference/motlp.md) using the `otlp` exporter.
+
+### Kubernetes deployment
+
+In Kubernetes, the host metrics receiver runs as part of the [DaemonSet collectors pipeline](../config/default-config-k8s.md#daemonset-collectors-pipeline) on every node to collect node-level host metrics at 60-second intervals. The receiver uses `root_path: /hostfs` to access the host's filesystem from within the container.
```

Comment:
```suggestion
In Kubernetes, the host metrics receiver runs as part of the [DaemonSet collectors pipeline](../config/default-config-k8s.md#daemonset-collectors-pipeline) on every node to collect node-level host metrics at 60-second intervals. The receiver uses `root_path: /hostfs` (a mounted Kubernetes volume) to access the host's filesystem from within the container.
```

I would add a note clarifing what "/hostfs" is, internally, is a Kubernetes volume mounted at the root filesystem "/" of the node/host.